invenio_rdm_records/config.py

```diff
@@ -352,7 +359,9 @@ def always_valid(identifier):
 ``RDM_PERSISTENT_IDENTIFIERS`` below)
 """
 
-
+# TODO find the logic that automatically removes the DOI if datacite_enabled 
```

Comment:
if this refers to the UI widget. This happens in the `customizations.py` [here](https://github.com/inveniosoftware/invenio-rdm-records/blob/281bcbbe53800ce819d52d5cf019d392b545fdc4/invenio_rdm_records/services/customizations.py#L12).